File: Results/DER/Translation/translation_output/w_test/gpt-4/codenet/java/python/p03477.py
Here is the Java code translated to Python, including handling the test input and providing the expected output:

```python
def main():
    numbers = input().split()
    left = int(numbers[0]) + int(numbers[1])
    right = int(numbers[2]) + int(numbers[3])
    
    if left > right:
        print("Left")
    elif left == right:
        print("Balanced")
    else:
        print("Right")

if __name__ == "__main__":
    main()
```

If you run this Python code with the test input '3 8 7 1', it will produce the expected output 'Left'.

File: Results/DER/Translation/translation_output/w_test/gpt-4/codenet/python/java/p03477.java
Below is the translated Java code that performs the same functionality as the provided Python code:

```java
import java.util.Scanner;

public class BalanceScale {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int A = scanner.nextInt();
        int B = scanner.nextInt();
        int C = scanner.nextInt();
        int D = scanner.nextInt();

        if (A + B < C + D) {
            System.out.println("Right");
        } else if (A + B == C + D) {
            System.out.println("Balanced");
        } else {
            System.out.println("Left");
        }
        
        scanner.close();
    }
}
```

You can run a test case in Java by compiling and running the `BalanceScale` class and then inputting the test case "3 8 7 1" on standard input. The expected output for this input would be "Left", as per the provided example.
